StudentData/classes/StudentData.cls

```diff
@@ -0,0 +1,24 @@
+public class StudentData {
+    	public StudentData(){
+		
+		//Insert Student Record
+		Student__c s1 = new Student__c();
+		s1.Name = 'ABC';
+        //Insert Student Record in Teacher Layout
+        s1.RecordTypeId = '0125i00000052gYAAQ';
```

Comment:
use Id recordTypeIdOfStudent = Schema.SObjectType.Student__c.getRecordTypeInfosByName().get('Name of record type').getRecordTypeId(); this one instead of hardcoded id.